Automattic/pocket-casts-ios
# EOY 2023: listened number story

| 📘 Part of: #1142 | 🎨 Designs: [Figma](https://www.figma.com/file/lH66LwxxgG8btQ8NrM0ldx/End-of-Year?type=design&node-id=1599-20385&mode=design) |
|:---:|:---:|

Adds the 2023 visual for the Listened Numbers Story and updates the order so it's the first story (after the intro).

**Animations are not part of this task and will be tackled later.**

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/62a7eec5-8f13-4b0d-9875-dc011e3969d2" width="300">

## DM Sans line spacing

We have an issue with the line spacing of DM Sans. Is just too big and SwiftUI doesn't allow setting a negative value. I'll leave as it is for now.

## To test

1. Make sure you're logged in into an account that has a few episodes listened
2. Go to Profile > Settings > Beta Features > enable `endOfYear`
3. Go back to Profile
4. Tap the EOY image
5. Skip to the second story
5. ✅ Check that the Listened Numbers story looks good
6. Tap "Share this story" > Save Image
7. Go to Photos
8. ✅ The story image asset should be the last one on Photos

Please test on a big device (iPhone 15 Pro Max, for example), normal device (iPhone 15), and a smaller device (iPhone SE).

## Checklist
